### PR TITLE
[EN/ZH] Add OFFC cirno tournament result sections

### PR DESCRIPTION
**Pull request by CloneWith**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/OFFC/cirno/en.md [hunk 1]
```diff
@@ -220,6 +267,26 @@ Match time: 21 September 2024 ~ 29 September 2024
...
+| 身为pp刷子因没练图而成了传说 | **Bingo!** | [#12](https://osu.ppy.sh/community/matches/115780911) |
+
+### Grand Finals
+
+Match time: 20 October 2024 ~ 2 November 2024
+
+| Team 1 | Team 2 | Match link |
+| --: | :-- | :-- |
+| **沟槽的烤翅还在开战舰追我** | Bingo! | [#13](https://osu.ppy.sh/community/matches/115914380) |
+| **这是什么** | 沟槽的烤翅还在开战舰追我 *(FF)* | - |
```
Comment (NaughtyChas) [inline wiki/Tournaments/OFFC/cirno/en.md:288]: Is it okay to use the abbreviation "FF" directly? Are there any other wiki articles like this?

File: wiki/Tournaments/OFFC/cirno/zh.md [hunk 2]
```diff
@@ -86,6 +86,15 @@ osu! FumoFumo Cup：琪露诺杯由众多的社区成员举办。
 | **#7** | 诟撡𤷙𣧏𡣀𠔑𡉄𫟺㻵𧂂𨾻𠪧 | ::{ flag=HK }:: [**doulikeplayosu**](https://osu.ppy.sh/users/25785799), ::{ flag=CN }:: [l2883119](https://osu.ppy.sh/users/8844222), ::{ flag=HK }:: [6dl_](https://osu.ppy.sh/users/30894174), ::{ flag=CN }:: [rabbit emperor](https://osu.ppy.sh/users/18753676), ::{ flag=JP }:: [madlife_AA](https://osu.ppy.sh/users/32901756), ::{ flag=CN }:: [alkali_earth](https://osu.ppy.sh/users/32858645), ::{ flag=CN }:: [hiesimo](https://osu.ppy.sh/users/31443365), ::{ flag=CN }:: [Cibo](https://osu.ppy.sh/users/9484743) |
 | **#8** | Helicopter | ::{ flag=TW }:: [**BocchiRocket**](https://osu.ppy.sh/users/20699990), ::{ flag=TW }:: [what th fxxl](https://osu.ppy.sh/users/25452354), ::{ flag=TW }:: [berrylove](https://osu.ppy.sh/users/16115679), ::{ flag=TW }:: [Explosion_MEOW](https://osu.ppy.sh/users/12806924), ::{ flag=TW }:: [yusen0806](https://osu.ppy.sh/users/33635384), ::{ flag=HK }:: [MyAngelNazuna](https://osu.ppy.sh/users/27141411), ::{ flag=CN }:: [\[mogezi\]](https://osu.ppy.sh/users/10483107), ::{ flag=TW }:: [puipui940515](https://osu.ppy.sh/users/20777456) |
 
+## 颁奖
```
Comment (NaughtyChas) [inline wiki/Tournaments/OFFC/cirno/zh.md:89]:
```suggestion
## 领奖台
```

"领奖台" would be better? The word "颁奖" used right here seems a bit weird (?

File: wiki/Tournaments/OFFC/cirno/zh.md [hunk 3]
```diff
@@ -202,6 +211,44 @@ osu! FumoFumo Cup：琪露诺杯由众多的社区成员举办。
   3. [EastNewSound feat. nayuta - Hiiro Gekka, Kyoushou no Zetsu -1st Anniversary Remix- [Extra Stage]](https://osu.ppy.sh/beatmapsets/1483410#osu/3042458)
   4. [Masayoshi Soken - In The Balance [Aglaia]](https://osu.ppy.sh/beatmapsets/1880199#osu/3870876)
 
+### 总决赛轮 (GF)
+
+**[在这里下载谱面包！（182 MB）](https://drive.google.com/file/d/15HH9owdVHT6LSYF1UzIWlNkeuMi0htDd/view)**
```
Comment (NaughtyChas) [inline wiki/Tournaments/OFFC/cirno/zh.md:216]:
```suggestion
**[在这里下载图包！（182 MB）](https://drive.google.com/file/d/15HH9owdVHT6LSYF1UzIWlNkeuMi0htDd/view)**
```

Any pre-written terms for `mappack`? We generally use "图包" in Chinese context.

File: wiki/Tournaments/OFFC/cirno/zh.md [hunk 4]
```diff
@@ -226,6 +273,26 @@ osu! FumoFumo Cup：琪露诺杯由众多的社区成员举办。
...
+| 身为pp刷子因没练图而成了传说 | **Bingo!** | [#12](https://osu.ppy.sh/community/matches/115780911) |
+
+### 总决赛轮 (GF Round)
+
+比赛时间：2024 年 10 月 20 日 ~ 2024 年 11 月 2 日
+
+| 队伍 1 | 队伍 2 | 比赛链接 |
+| --: | :-- | :-- |
+| **沟槽的烤翅还在开战舰追我** | Bingo! | [#13](https://osu.ppy.sh/community/matches/115914380) |
+| **这是什么** | 沟槽的烤翅还在开战舰追我 *(FF)* | - |
```
Comment (NaughtyChas) [inline wiki/Tournaments/OFFC/cirno/zh.md:294]: Same before.

> Is it okay to use the abbreviation "FF" directly? Are there any other wiki articles like this?

File: wiki/Tournaments/OFFC/cirno/zh.md (hunk 3 above)
Comment (CloneWith) [reply wiki/Tournaments/OFFC/cirno/zh.md]: The latter seems better, and would be put in use.

File: wiki/Tournaments/OFFC/cirno/zh.md (hunk 2 above)
Comment (CloneWith) [reply wiki/Tournaments/OFFC/cirno/zh.md:89]: How about `领奖信息`? It's a bit long probably, but `颁奖` or `领奖台` lacks something I think...

Comment (NaughtyChas) [reply wiki/Tournaments/OFFC/cirno/zh.md:89]: > How about `领奖信息`? It's a bit long probably, but `颁奖` or `领奖台` lacks something I think...

Yeah. Also we missed the 3rd place  `Bingo!`.

Comment (NaughtyChas) [reply wiki/Tournaments/OFFC/cirno/zh.md]: Resolved.

**Review by TicClick — APPROVED** [review]

**Pull request merged**